### DISCUSSION TITLE
Supported Chocolatey

### DISCUSSION THREAD

**Discussion by pit-ray** [pit-ray/win-vind#35] [News]
Uploaded to Chocolatey upon request. 

[community.chocolatey.org/packages/win-vind](https://community.chocolatey.org/packages/win-vind)

To install it, please run the following command.
```sh
$ choco install win-vind
```

Thanks.

**Comment by Positron010** [comment]
Hi,
That's great. Thanks for your hard work. Consider also adding it to [winget](https://github.com/microsoft/winget-cli).

Thanks

**Reply by pit-ray** [reply]
I don't know if it will be approved, but I will try to submit it in the near future.

**Reply by pit-ray** [reply]
win-vind has been approved by winget. 🎉 (#36)  
Thanks.
